src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -59,6 +59,7 @@ static const TypeMapping dnnlFCTypeMapping {
     // quantization configuration
     // int8 inner_product does not support f16 output and bias
     {{_u8 | _i8, _i8, _any, _f16},                            pt(bypass(), bypass(), just<f32>(), just<f32>())},
+    {{_u8 | _i8, _i8, _f16, _any},                            pt(bypass(), bypass(), just<f32>(), bypass())},
```

Comment:
Does oneDNN prim really supports any precision on the output?

Reply:
Hi, @dmitry-gorokhov :  the initial purpose of this change is to fix some Int8 models(e.g. INT8/resnet) fail with log 'could not create a primitive descriptor for an inner product forward propagation primitive'. 
And based on your concern, since it's for {_u8 | _i8, _i8, _f16,..} cases and its bias will be enforced to f32, how about we also set just<f32>() for output?

Reply:
Forcing f32 ouput precision might negatively affect the perf. Please provide details which precision have Conv on its inputs/outputs (the one which you fixed by this change).

Reply:
The Node info in origin ir is:
![1](https://github.com/openvinotoolkit/openvino/assets/56590087/d3534810-3700-4d23-a8c8-e27f4205d799)
in dumped graph is:
![2](https://github.com/openvinotoolkit/openvino/assets/56590087/2bf51909-940c-4b17-a911-2d1cdeedfcc4)
so, for this single testcase, seems  set just< f32 >() for output could keep algin with previous logic

and the error log is: 
![3](https://github.com/openvinotoolkit/openvino/assets/56590087/cbb6ed30-4102-4f46-b0d6-bd5c4abb0f27)

Reply:
Please take a look on this comment: https://github.com/openvinotoolkit/openvino/pull/24315#discussion_r1607650678. I think we need to change all quantized FC configs. The idea is to descibe suppoted patterns in one config and have fallback config for all other cases. Should look like this:
```
{{_u8 | _i8, _i8, _u8 | _i8 | _i32 | _bf16 | _f32, _u8 | _i8 | _i32 | _bf16 | _f32}, pt(bypass(), bypass(), bypass(),  bypass())},
{{_u8 | _i8, _i8, any, any}, pt(bypass(), bypass(), just<f32>(), just<f32>())},
```
So whatever precision we support on the plugin level (fp16, fp8, fp64) it will be correctly handled by the second line. All cases which are supported by oneDNN are described in the first line and has higher priority for dispatch.

Reply:
Done